stream_data.py

```diff
@@ -0,0 +1,15 @@
+from gen_temperature_stream import *
```

Comment:
1. Better to give the file name, that indicates the functionality(ex: sender)
2. There is no test written for send to console logic.